README.md

```diff
@@ -26,6 +28,34 @@ To start pi-blaster and have it relaunched automatically on every reboot:
 
     sudo make install
 
+### Build a deb package and install with dpkg
+
+You may prefer to package pi-blaster first to better integrate with Debian system.
+
+#### Raspbian 7 ("Wheezy") users
+
+This is the most common case. If you are not sure which version of Debian you are
+using, follow this instruction.
+
+    sudo apt-get install debhelper dh-autoreconf dpkg-dev autoconf
+    cd /path/to/pi-blaster/
+    dpkg-buildpackage -us -uc -i
+    sudo dpkg -i ../pi-blaster*.deb
+
+#### Raspbian 8 ("Jessie") users
+
+If you have upgraded your Raspberry Pi to use Raspbian 8 with systemd, you may
+still follow the instruction given for Wheezy users. However, an extra
+special step may help you to better integrate with systemd.
+
+    sudo apt-get install debhelper dh-autoreconf dh-systemd dpkg-dev \
+         init-system-helpers autoconf
+    cd /path/to/pi-blaster/
+    git revert 656e14c6acbb77aec95a57783726c043ee642f34
```

Comment:
This is a difficult problem: some people are still using Wheezy (which is `oldstable`, because default Raspbian wasn't upgraded to Jessie until September 2015), and Wheezy has neither `dh-systemd` nor `init-system-helpers` package. If the default debian rule is written to add systemd support (which is Debian's standard solution to the coexistence between systemd and sysvinit on Jessie), those users will need to **manually** enable `debian-backports` for those two packages that was backported to Wheezy even though they are not really using systemd. Maybe it's not a good idea either.

Using `git revert` is indeed not very good-looking. Will a wrapper script (e.g. `autobuild`) and symlink be better?